packages/flutter/lib/src/widgets/routes.dart

```diff
@@ -1141,11 +1155,28 @@ abstract class ModalRoute<T> extends TransitionRoute<T> with LocalHistoryRoute<T
     return child;
   }
 
+  /// Whether this route should ignore pointers when it is the current route and
+  /// transitions are in progress.
```

Comment:
nit: maybe skip the `when it is the current route` part, your second paragraph explains the isCurrent pretty well.

Reply:
Got it—done